src/interface/src/app/plan/plan.component.html

```diff
@@ -1,14 +1,19 @@
 <div class="root-container">
   <app-plan-unavailable *ngIf="planNotFound"></app-plan-unavailable>
   <ng-container *ngIf="!planNotFound && plan">
-    <div class="plan-summary-panel" *ngIf="currentPlanStep === PlanStep.Overview">
+    <router-outlet #outlet="outlet"></router-outlet>
```

Comment:
I like this approach of navigating in the PlanComponent. We should make the plan overview and scenario planning flow routable too. (No need to do that in this change.)